ansible/monitoring-monasca-container.yml

```diff
@@ -3,8 +3,7 @@
 #
 ---
 - name: Deploy Monasca Swarm monitoring service
-  hosts: localhost
-  connection: local
-  gather_facts: no
+  hosts: master
```

Comment:
Does this group always exist?

Reply:
In the container-infra role, yes it does :)

Reply:
Do we need to use this playbook for non container clusters?

Reply:
Not this one, there is a separate playbook for deploying to non-container clusters.